examples/binaries/calc-hash/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
this one should be always no_std, due to representing just some common, shared entities